CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ x.y.z Release notes (yyyy-MM-dd)
 Upgraded realm-core from ? to ?
 Upgraded realm-sync from ? to ?
 
+4.3.2 Release notes (2020-02-04)
+=============================================================
+### Enhancements
+* Fix for memory leak caused by `property_copyAttributeList`
+
+### Fixed
+* property_copyAttributeList was not being released after being assigned to a local variable. 
```

Comment:
The changelog is targeted at developers using Realm, and should focus on what symptoms they'd see rather than the exact change made to fix it. For example, here I'd write something like "Fix a memory leak attributed to `property_copyAttributeList` the first time a Realm is opened when using Realm Swift. ([#6409](https://github.com/realm/realm-cocoa/issues/6409), since 4.0.0)." The main thing a person reading the changelog wants to know is if upgrading to the new version will fix a problem they're hitting; if they want to know more about the specific details of the change then the link to the issue helps them find that.

Reply:
@tgoyne noted and updated with your feedback.